EFCore.InMemory.HierarchyId/EFCore.InMemory.HierarchyId.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <AssemblyName>EntityFrameworkCore.InMemory.HierarchyId</AssemblyName>
+    <RootNamespace>Microsoft.EntityFrameworkCore.InMemory</RootNamespace>
+    <Authors>Brice Lambson, Adrian Paul Nutiu, et al.</Authors>
```

Comment:
cc: @bricelam

Reply:
I didn't author anything here... 😏 Fine with just *Adrian Paul Nutiu* (the first person to modify it can add *et al.*)

Reply:
Updated in https://github.com/efcore/EFCore.SqlServer.HierarchyId/pull/33/commits/ce705ae01aa4d132ab00345ac6c54e4681a2b207